Project-3/README.md

```diff
@@ -11,67 +11,238 @@ For any question involving the use of Protege, please be sure to import:
   (b) Chair of the UB Philosophy Department
+    Role
   (c) SARS-CoV-2
+    Object
   (d) Mexico City
+    I prefer to choose Site over Object Aggrogate. But 2D Spatial Region is the best choice.
   (e) The trunk of a minivan
+    Fiat Object Part
   (f) Occupation
+    Role
   (g) Ocean
+    Site is my top choice since the water level of the ocean is not of our concern, but to be safe, a 3D Spatial Region may be a good option as well. I am resisting to pick Object Aggrogate as an option
   (h) Lake
+    For the same reasons above, I will prefer to choose Site or 3D Spatial Region. A lake can go dry and will still be considered as a lake (Salt lake and Urumieh lakes in Iran)
+
 ```
 
 7. True or False; explain your answers:
 ```
   (a) An instance of Material Entity can have an instance of Immaterial Entity as part.
+    False. By definition, according to the axioms in BFO, Material Entity and Immaterial Entity are disjoint. Thus, there cannot be a single part of a material entity which an immaterial entity.
   (b) An instance of Immaterial Entity can have an instance of Material Entity as part.
+    False. According to the same axioms as mentions above, a Material Entity is disjoint with "part of some Material Entity" according to BFO.
   (c) An organization may have another organization as part.
-  (d) An organization may have no members as part. 
+    True. The organization United States Government has the Organization Department of Aggroculture as its part.
+  (d) An organization may have no members as part.
+    True. A club can lose its all members and regain some members in the next year. (of course one could argue whether it is the same club or not)
   (e) Any site is partially bounded by some instance of Material Entity.
-  (f) A book placed under the leg of a wobbly table has acquired a new function. 
+    Flase. By the BFO defintion of Site, it must be wholy or partially bounded by a material entity or may be a three-dimensional immaterial part of an immaterial entity (partially or wholly) bounded by a material entity. 
+  (f) A book placed under the leg of a wobbly table has acquired a new function.
+    False. Unless the book was an artifact made solely for the purpose of being used as described, functions are realizable entities that an object is precisely made or evolved for that purpose.
   (g) A glass vase cushioned with packing material for all time, has the disposition to break. 
+    True. Disposition is relavent to the physicality of the object and not its environment.
   (h) Spacetime is a class in BFO.
+    False. I say false, just because it is technicaly Spatiotemporal Region that is a BFO class.
   (i) The continuant fiat boundary class of BFO is closed, meaning, there are no subclasses beyond those identified presently in BFO. 
+    True. This is with the assumption that we are talking about adding subclasses in BFO. It might be the case that a field-specific ontology would want to add subclasses.
```

Comment:
False. As its entry in BFO file says, Continuant fiat boundary doesn't have a closure axiom because the subclasses don't necessarily exhaust all possibilites. An example would be the mereological sum of two-dimensional continuant fiat boundary and a one dimensional continuant fiat boundary that doesn't overlap it.